docs/index.txt

```diff
@@ -35,6 +35,8 @@ following pages should help you get started:
 
 - :doc:`/tutorial/gridfs`
 
+- :doc:`tutorial/persistable-classes`
```

Comment:
Shouldn't this be `/tutorial...` for consistency with the other links?

Reply:
That's correct, fixed.